src/components/settings-modal/settings-modal.js

```diff
@@ -209,13 +251,17 @@ export const mapStateToProps = (state) => ({
   flags: state.flags,
   showFeatureHints: state.showFeatureHints,
   isPrettyName: state.isPrettyName,
+  showDatasetPreviews: state.preferences.showDatasetPreviews,
```

Comment:
It should be like this right?
`showDatasetPreviews: state.showDatasetPreviews,` 
or I am missing something ?

Reply:
I tested locally and surprisingly it works 🗡️  .... not sure if the prop is used to set the local state of the toggle

Reply:
This just keeps `showDatasetPreviews` nested within the preferences state its just to keep it more distinct incase in the future we want to add more preferences. We can also go with the original global state version as well.

Reply:
it is good to have a nesting, but I guess the question is how showDatasetPreviews is under preferences when the initial state does not have a nesting